shell/browser/api/electron_api_web_contents.cc

```diff
@@ -2341,11 +2346,16 @@ void WebContents::SendInputEvent(v8::Isolate* isolate,
       v8::Exception::Error(gin::StringToV8(isolate, "Invalid event object")));
 }
 
-void WebContents::BeginFrameSubscription(gin_helper::Arguments* args) {
+void WebContents::BeginFrameSubscription(gin::Arguments* args) {
   bool only_dirty = false;
   FrameSubscriber::FrameCaptureCallback callback;
 
-  args->GetNext(&only_dirty);
+  if (args->Length() > 1) {
+    if (!args->GetNext(&only_dirty)) {
+      args->ThrowError();
```

Comment:
Descriptive error message ?

Reply:
I think this will show "conversion failure at index 0" or something, which is consistent with other APIs (if not exactly the most useful)